src/gui/src/LogWindowBase.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
+    <width>996</width>
```

Comment:
Are these changes intended? Sadly when you edit ui files the designer loves to changes other lines you didn't want to on you.

To avoid editing this file at all in the constructor after `setupUi(this)` you can call  `m_LogOutput->setMaximumBlockSize(10000)`